def raise_error(error_message: str = "We experienced an error!"):
    raise SystemExit(error_message)
